src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx

```diff
@@ -153,7 +182,7 @@ Global decisions cover a variety of correlation scenarios. Review descriptions t
       </td>
 
       <td>
-        Correlation activated because custom application name, policy and custom id are the same respectively. This is to reduce application incidents, particularly cater to custom tag users. Learn more about [tags](/docs/new-relic-solutions/new-relic-one/core-concepts/use-tags-help-organize-find-your-data/). Customed tag id could be defined by condition family id or other id values used as a key to identify connections between data.
+        Correlation activated because custom application name, policy and custom id are the same respectively. This is to reduce application issues, particularly cater to custom tag users. Learn more about [tags](/docs/new-relic-solutions/new-relic-one/core-concepts/use-tags-help-organize-find-your-data/). Customed tag id could be defined by condition family id or other id values used as a key to identify connections between data.
```

Comment:
maybe 'correlation was activated'

serial comma after 'policy'

capitalize 'ID'

grammar issue with 'particularly cater...'

For links, we usually try to make doc description more explicit, so like 'For more about tags, see [Tags].'

'customed' typo

capitalize 'ID' instances

Reply:
done